openmetadata-docs/content/v1.4.x/connectors/dashboard/superset/yaml.md

```diff
@@ -185,6 +186,14 @@ source:
 
 {% /codePreview %}
 
+## Securing Superset Connection with SSL in OpenMetadata
+
+To establish secure connections between OpenMetadata and Superset, in the `YAML` under `sslConfig`, we need to add `caCertificate` and update the certificate path. Ensure that the certificates are accessible from the Airflow Server.
+
+```yaml
+    sslConfig:
+      caCertificate: /path/to/cacert.crt
+```
```

Comment:
For SSL within Superset, let's go ahead with bifurcating for mysql and postgres which are used within Superset Connection